packages/app/src/providers/DataProvider.tsx

```diff
@@ -43,25 +43,23 @@ function DataProvider(props: PropsWithChildren<Props>) {
   const { api, children } = props;
 
   const entitiesStore = useMemo(() => {
-    const childCache = new Map<string, Entity>();
+    const childCache = new Map<string, Exclude<ChildEntity, Group>>();
```

Comment:
Wait, why don't we store group metadata ? Does this mean that we need to fetch again when asking for a group path ?

Reply:
Groups (with children) are stored in the `entitiesStore`'s internal cache. We don't want to store child groups without children in the `childCache`, because if we did, we would end up with groups without children when calling `entitiesStore.get()`... and we definitely don't want that.